mediaqueries-4/Overview.bs

```diff
@@ -1733,19 +1733,18 @@ max brightness or low brightness as it relates to displays. As such, the
 determination for '@media/dynamic-range' and '@media/video-dynamic-range' will
 be defined by the user agent.
 
-<h3 id="video-prefixed-properties">
-Video Prefixed Properties scenarios</h3>
+<h3 id="video-prefixed-properties">Video Prefixed Feature</h3>
```

Comment:
Add 's' to end of Feature (sorry, my typo). Change id to match: id="video-prefixed-features".